src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -340,309 +334,235 @@
-                    top: 0;
-                }
-            }
-        }
-
-        .datagrid-foot-select.checkbox {
-            display: block;
-            line-height: inherit;
-            flex: 1 1 (1.5rem - (2 * $clr-rem-1px)); // can't use calc() in flex. #thanksIE!
-            margin-right: 0.083333rem; // Space for the instance when a hideable column toggle is next to it.
-
-            label {
-                color: unset;
-                cursor: default;
-                opacity: 1;
-                line-height: inherit;
-            }
-
-            input[type="checkbox"] + label {
-                &::before, &::after {
-                    top: 0.333333rem;
-                }
-
-                &::after {
-                    border-left-color: $clr-white;
-                    border-bottom-color: $clr-white;
-                }
-            }
-        }
-
-        .datagrid-row-actions {
-            padding: 0;
-
-            clr-icon {
-                height: $clr-table-lineheight;
-                vertical-align: bottom;
-            }
-        }
-
-        .datagrid-signpost-trigger .signpost {
-            margin: -0.3rem 0;
-            height: 1.03rem;
-            .signpost-trigger {
-                height: inherit;
-                line-height: 1rem;
-            }
-        }
-
-        $horizontalPadding: ($clr-datagrid-fixed-column-size - 1.25rem) / 2;
-
-        .datagrid-expandable-caret {
-            padding: calc(0.125rem - #{$clr-default-borderwidth}) $horizontalPadding 0.125rem;
-            text-align: center;
-
-            .datagrid-expandable-caret-button {
-                @include clr-no-styles-button();
-                cursor: pointer;
-                height: 1.25rem;
-                width: 1.25rem;
-            }
-
-            .datagrid-expandable-caret-icon {
-                color: $gray-dark;
-                // nudge down for vertical alignment...
-                margin-top: 0.125rem;
-                svg {
-                    transition: transform 0.2s ease-in-out;
+                    .close {
+                        float: none;
+                    }
+                }
+
+                // FIXME: remove
+                .datagrid-filter-apply {
+                    margin-bottom: 0;
                 }
             }
 
-            .spinner {
-                margin-top: 0.25rem;
+            &.datagrid-fixed-width {
+                flex: 0 0 auto;
             }
 
-            &.datagrid-column {
-                padding: $clr-table-topcellpadding $clr-table-cellpadding $clr-table-bottomcellpadding;
+            &--hidden {
+                display: none;
             }
-        }
 
-        .datagrid-body .datagrid-row {
-            border-top: $clr-table-borderwidth solid $clr-table-border-color;
-            flex-shrink: 0;
+            &.datagrid-fixed-column {
+                flex: 0 0 $clr-datagrid-fixed-column-size;
+                // IE10 needs this
```

Comment:
Again, IE10?

Reply:
Will remove the comment. We need the min/max widths set.